ddr/ddr/GameScene.swift

```diff
@@ -31,12 +76,12 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
   
   var velocity : CGFloat = 0.02
   
-  let audioManager = AudioManager()
+  
   let motionManager = CMMotionManager()
   
   var lives = [SKSpriteNode]();
   
-  var bats = [Bat]()
+//  var bats = [Bat]()
```

Comment:
Are you commenting these lines out because they'll likely be used again/you're not sure of the changes?

Otherwise I'd just go ahead and delete them

Reply:
I am fixing this in my next commit